extensions/http/src/main/java/org/mule/extension/http/internal/request/validator/HttpRequesterProvider.java

```diff
@@ -73,7 +73,7 @@
    */
   @Parameter
   @Optional
-  @Placement(group = URL_CONFIGURATION, order = 2)
+  @Placement(group = CONNECTION, order = 2)
```

Comment:
Maybe is better if the order of this fields follows the same order defined in `@Placement`